### PR TITLE
Release notes for week ending 27 Jan 2017

### PR DESCRIPTION
**Pull request by srpearce**
## [DOC-3559](https://openedx.atlassian.net/browse/DOC-3559)

Add a description of your changes with links to any relevant material.

### Date Needed (optional)

31 Jan 2017

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [x] Subject matter expert: @marcotuts 
- [ ] Subject matter expert: @sstack22 
- [x] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: en_us/release_notes/source/2017/2017-01-27.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#################################
+Week Ending 27 January 2017
+#################################
+
+The following information summarizes what was released in the edX platform during the week ending 20 January 2017.
```
Comment (catong) [inline en_us/release_notes/source/2017/2017-01-27.rst]: Need to update to 27 January

File: en_us/release_notes/source/2017/analytics/analytics_2017-01-27.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+* A problem that prevented some ``play_video`` events from appearing after
```
Comment (catong) [inline en_us/release_notes/source/2017/analytics/analytics_2017-01-27.rst]: Perhaps "prevented... logging of events" vs "prevented from appearing"?

File: en_us/release_notes/source/2017/lms/lms_2017-01-27.rst [hunk 3]
```diff
@@ -0,0 +1,11 @@
+* Course teams can now preview content according to content group. For more
```
Comment (catong) [inline en_us/release_notes/source/2017/lms/lms_2017-01-27.rst]: Suggest adding "draft" to clarify the publishing state of content that is previewed: "can now preview draft content"

File: en_us/release_notes/source/2017/lms/lms_2017-01-27.rst [hunk 4]
```diff
@@ -0,0 +1,5 @@
+* For course discussions, options to sort posts by most activity, recent
+  activity, or most votes are now available for discussions in the body of the
```
Comment (catong) [inline en_us/release_notes/source/2017/lms/lms_2017-01-27.rst]: We refer to them as "inline discussions" in doc - would be good to use the same terminology here

**Review by catong — COMMENTED** [review]
A few comments

**Comment by marcotuts** [comment]
👍

**Pull request merged**